### PR TITLE
Mark DOMException as serializable

### LINKED ISSUE #729: Define serialization and deserialization steps for DOMException
See https://github.com/whatwg/html/issues/4268 for context.

cc: @domenic

### PR DESCRIPTION
**Pull request by yutakahirano**
Mark DOMException as serialiable

Also define serialization and deserialization steps.
tests: https://github.com/web-platform-tests/wpt/pull/17095

Fixes #729

***
<a href="https://pr-preview.s3.amazonaws.com/yutakahirano/webidl/pull/732.html" title="Last updated on Jun 7, 2019, 7:51 AM UTC (0017a1d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/heycam/webidl/732/4ef3509...yutakahirano:0017a1d.html" title="Last updated on Jun 7, 2019, 7:51 AM UTC (0017a1d)">Diff</a>

### CODE REVIEW

**Comment by yutakahirano** [comment]
PTAL

File: index.bs [hunk 1]
```diff
@@ -13409,6 +13410,20 @@ The <dfn attribute for="DOMException"><code>code</code></dfn> attribute's getter
 legacy code indicated in the [=error names table=] for this {{DOMException}} object's
 [=DOMException/name=], or 0 if no such entry exists in the table.
 
+DOMException objects are [=serializable objects=].
```
Comment (domenic) [inline index.bs]:
```suggestion
{{DOMException}} objects are [=serializable objects=].
```

**Comment by bzbarsky** [comment]
Note the problem described in https://github.com/whatwg/html/issues/4268#issuecomment-497388674

File: index.bs [hunk 2]
```diff
@@ -13397,7 +13397,7 @@ Each {{DOMException}} object has an associated <dfn for="DOMException">name</dfn
 The <dfn constructor for="DOMException"><code>DOMException(|message|, |name|)</code></dfn>
 constructor, when invoked, must run these steps:
 
-1. Set <b>this</b>'s [=DOMException/name=] to |name|.
+1. Set <b>this</b>'s [=DOMExcep tion/name=] to |name|.
```
Comment (CYBAI) [inline index.bs]: Maybe this space is added accidentally 👀?

Comment (yutakahirano) [reply index.bs]: Thanks! Fixed.

**Comment by yutakahirano** [comment]
Added sentences for `.stack`.

File: index.bs [hunk 3]
```diff
@@ -13409,6 +13410,25 @@ The <dfn attribute for="DOMException"><code>code</code></dfn> attribute's getter
 legacy code indicated in the [=error names table=] for this {{DOMException}} object's
 [=DOMException/name=], or 0 if no such entry exists in the table.
 
+{{DOMException}} objects are [=serializable objects=].
+
+Their [=serialization steps=], given <var>value</var> and <var>serialized</var>, are:
+<ol>
```
Comment (TimothyGu) [inline index.bs]: Nit: extra new line above.

Comment (yutakahirano) [reply index.bs]: Done.

**Pull request merged**